modules/age.nix

```diff
@@ -228,7 +228,7 @@ in {
     identityPaths = mkOption {
       type = types.listOf types.path;
       default =
-        if (config.services.openssh.enable or false)
+        if ((config.services.openssh.enable or false) == true && config.services.openssh ? hostKeys)
```

Comment:
I think I would consider inverting the logic and thereby avoid messing with the linux logic:

```
if isDarwin then [ darwin values ]
else if (config.services.openssh.enable or false) then linux-stuff
else []
```

If `hostKeys` ever makes an appearance on darwin this can be refactored, but otherwise this makes things a little more complex and less readable for no gain. (And also [causes a false-positive statix lint](https://github.com/oppiliappan/statix/issues/79#issuecomment-2585773951).)